apps/web/lib/env.ts

```diff
@@ -19,23 +19,26 @@ export const env = () =>
 
       UNKEY_API_URL: z.string().url().default("http://127.0.0.1:8080"),
       NEXT_PUBLIC_UNKEY_API_URL: z.string().url().default("http://127.0.0.1:8080"),
-      UNKEY_APP_AUTH_TOKEN: z.string(),
+      UNKEY_APP_AUTH_TOKEN: z.string().min(1),
 
       CLERK_WEBHOOK_SECRET: z.string().optional(),
       RESEND_API_KEY: z.string().optional(),
       RESEND_AUDIENCE_ID: z.string().optional(),
 
       UPTIME_CRON_URL_COLLECT_BILLING: z.string().optional(),
       PLAIN_API_KEY: z.string().optional(),
+
+      NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY: z.string().min(1),
+      CLERK_SECRET_KEY: z.string().min(1),
     })
     .parse(process.env);
 
 export const dbEnv = () =>
   z
     .object({
-      DATABASE_HOST: z.string(),
-      DATABASE_USERNAME: z.string(),
-      DATABASE_PASSWORD: z.string(),
+      DATABASE_HOST: z.string().min(1),
```

Comment:
A min length of 1 is required when parsing, because an empty `.env` variable does not throw an error.